src/main/resources/site/parts/searchResult/searchResult.ts

```diff
@@ -112,15 +116,39 @@ export function renderPart(req: Request): React4xpResponse {
 
     let bestBetResult: PreparedSearchResult | null
     if (firstBet && (firstBet.constructor !== Array)) {
+      let date: string = firstBet.data.linkedContentDate
+      let title: string | undefined = firstBet.data.linkedContentTitle
```

Comment:
Suggestion: Remove undefined type, make it something like this, moving the logic from the bestBetResult object to the variable instantiation? 
`let title: string = firstBet.data.linkedContentTitle ? firstBet.data.linkedContentTitle : ''`